memory-store.js

```diff
@@ -6,6 +6,21 @@ var compareTime = require('./compare-time')
  * It is good for tests, but not for server or client usage,
  * because it doesn’t save events to file or localStorage.
  *
+ * Think about this store as a basic store realization.
+ *
+ * Every Store class should provide three standard methods:
+ * add, get and remove.
+ *
+ * Every Store for client should provide two more methods:
+ * getSynced, setSynced.
+ *
+ * @param {object} [options] Store options.
+ * @param {number} [options.synced=0] Events with lower `added` time in current
```

Comment:
Looks good. But `current` not looks so clear. `Sync#currentProtocol` looks like a protocol version of current connection. Also `other` prefix is not good too.

Maybe there are a special terms for it in other protocol docs?